Update managing-multiple-accounts.md (#34098)
github/docs: content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/managing-multiple-accounts.md

{% data reusables.git.confirm-credential-manager %}
{% data reusables.git.clear-the-stored-credentials %}
   {% data reusables.git.no-credential-manager %}

>    * If the output is `osxkeychain`, you're using the macOS keychain. To clear the credentials, enter the following command and interactive inputs:

Reviewer: ```suggestion
   * If the output is `osxkeychain`, you're using the macOS keychain. To clear the credentials, you can use the credential helper on the command line:
```

     ```shell copy

>      > git credential-osxkeychain erase
>      host=github.com
>      protocol=https
>      ENTER

Reviewer: ```suggestion
     ```suggestion
     $ git credential-osxkeychain erase
     host={% data variables.product.product_url %}
     protocol=https
     > [Press Return]
```

{% data reusables.git.confirm-credential-manager %}
{% data reusables.git.clear-the-stored-credentials %}
   {% data reusables.git.no-credential-manager %}
   * If the output is `osxkeychain`, you're using the macOS keychain. To clear the credentials, enter the following command and interactive inputs:

>      ```shell copy

Reviewer 2: also when mixing prompt and output, the whole example should not be copied perhaps?
```suggestion
     ```shell
```

>      ```suggestion
>      $ git credential-osxkeychain erase
>      host={% data variables.product.product_url %}
>      protocol=https
>      > [Press Return]
>      >

Reviewer 2: fyi the suggestion got somehow applied wrong, this needs a little edit:

```suggestion
     $ git credential-osxkeychain erase
     host={% data variables.product.product_url %}
     protocol=https
     > [Press Return]
     >
```
